backend/de.metas.externalsystem/src/main/sql/postgresql/system/80-de.metas.externalsystem/5645370_sys_gh13198_External_System_Auth_Token.sql

```diff
@@ -0,0 +1,83 @@
+-- 2022-06-28T13:55:04.441Z
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541464,'S',TO_TIMESTAMP('2022-06-28 16:55:04','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','EXTERNAL_SYSTEM_AUTHORIZATION_TOKEN',TO_TIMESTAMP('2022-06-28 16:55:04','YYYY-MM-DD HH24:MI:SS'),100,'2ac8d2ac2c0d485c970db5d96f89d809')
+;
+
+-- 2022-06-30T14:59:46.179Z
+UPDATE AD_SysConfig SET ConfigurationLevel='S', Description='This is a sys config with an authorization token that is used to check that metasfresh has the appropriate authorization token.', Name='de.metas.externalsystem.externalservice.authorization.authToken',Updated=TO_TIMESTAMP('2022-06-30 17:59:46','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_SysConfig_ID=541464
```

Comment:
i don't unserstand the description. how is this value used? is is set to some `AD_User_AuthToken` value ? if yes - to which? (metasfresh / AD_USer_ID=100 ?) 

also, please 
- outline the relation between this value and the auth-token in `application.properties`
- if we don't remove the token properties from `application.properties` (why not?), then please also add an informative comment there

Reply:
@metas-ts yes, right now it is it is set to an AD_User_AuthToken value, which, as per the prerequisites of the documentation is called "local-dev". I'm not sure which user is that assigned to.

Also, there is no relation between this value and the auth-token from application.properties. The value from application.properties is only used when sending a request to Metasfresh, if the auth-token is not set from the newly added logic.

@cp-ps @metas-ts should we remove it from application.properties

Reply:
hi @mi-ps 
>  I'm not sure which user is that assigned to.

ok, then please find out how your feature works :-), and document it

> should we remove it from application.properties?

I understand from talks with cosmin that externalSystems will try getting a token from MF until it receives one. So, yes, please remove it. it's generating FUD.